### PR TITLE
Escape `&` for real this time

### PR DESCRIPTION
**Pull request by erik-stein**
Previous fix (#18) didn't really work, now it is fixed.

### CODE REVIEW

File: main/tex_compiler/tex_compiler.py
Comment (GitThisJoel) [inline main/tex_compiler/tex_compiler.py:1]: I did some digging and found that `"...".replace("&", r"\&")` might work instead. I think it is easier to read.

```python
import sys

out_file = "out"
with open(out_file, "w") as f:
    for line in sys.stdin: 
        f.write(line.replace('&', r'\&'))
```

with an in-file like:
```
&
&&
hej && hej
& bar &
```

yields
```
\&
\&\&
hej \&\& hej
\& bar \&
```

**Review by GitThisJoel — APPROVED** [review]
See my comment, do what you consider to be the best option and merge the branch.

File: main/tex_compiler/tex_compiler.py
Comment (erik-stein) [reply main/tex_compiler/tex_compiler.py:1]: Seems like this is not needed, i guess because `\&` is not a valid escape character.

**Review by erik-stein — COMMENTED** [review]
I think I found the underlying issue, we already escape some characters in `zettle_parser.py`, so they got escaped twice.

I moved the `replace` calls to `tex_compiler.py` because I believe that is is the tex compiler's responsibility to format the strings for correct output.

**Pull request merged**